editor/components/block-list/index.js

```diff
@@ -100,7 +101,21 @@ class BlockList extends Component {
 		}
 	}
 
+	/**
+	 * Handles a pointer move event to update the extent of the current cursor
+	 * multi-selection.
+	 *
+	 * @param {MouseEvent} event A mousemove event object.
+	 *
+	 * @returns {void}
+	 */
 	onPointerMove( { clientY } ) {
+		// We don't start multi-selection until the mouse starts moving, so as
+		// to avoid dispatching multi-selection actions on an in-place click.
+		if ( ! this.props.isMultiSelecting ) {
```

Comment:
Maybe a stupid question, but could this be build in the reducer if it should always be used like this? Same for stopping the selection.

Reply:
>Maybe a stupid question, but could this be built in the reducer if it should always be used like this? Same for stopping the selection.

Can you clarify what you mean? I have two interpretations:

- Should the `state.blockSelection.isMultiSelecting` flag be automatically assigned to `true` when `MULTI_SELECT` is dispatched, vs. an explicit `START_MULTI_SELECT` action?
   - Perhaps, if we're okay with the fact that `isMultiSelecting` is not reflected immediately upon starting to drag the mouse, but only after the drag extends across at least one additional block
   - I'm also not sure how it would be possible to infer the end of a multi-selection via this behavior
- Handle the check to only change the value of `isMultiSelecting` if it's not already `true`, from the reducer itself, rather than in the component
   - My initial reaction is: Yes, and it should probably be doing so anyways, but that as it stands, we need the component to check due to #4593

Reply:
Yes, I mean the second point. Sounds good to me then. :)

Reply:
I added 6d6231e to cover our bases 😄